### PR TITLE
Add obs-package-status service to docker compose file

### PR DESCRIPTION
**Pull request by danidoni**
This PR:
* Adds the `obs-package-status` service to the `docker-compose.yml`.
* Apply known best practices
* Complements https://github.com/openSUSE/obs-tools/pull/37

### CODE REVIEW

**Comment by danidoni** [comment]
[create `docker-compose` configuration to all `obs-tools` tools](https://trello.com/c/iVBqVj7O/1353-create-docker-compose-configuration-to-all-obs-tools-tools)

**Comment by vpereira** [comment]
@danidoni  I don't understand the title. The tool is already dockerized.

**Comment by danidoni** [comment]
> @danidoni I don't understand the title. The tool is already dockerized.

You are right. Changed

**Comment by danidoni** [comment]
> Please describe in the PR what you are really doing. AFAICS you are adding a service `open-package-status` to the `docker-compose` and adapting the existing `Dockerfile` for it, no?

Updated the PR description. Check again, please.

**Comment by vpereira** [comment]
> Updated the PR description. Check again, please.

please fix the `docker-compose.yml` conflict

**Comment by danidoni** [comment]
> > Updated the PR description. Check again, please.
> 
> please fix the `docker-compose.yml` conflict

Fixed

File: docker-compose.yml [hunk 1]
```diff
@@ -18,3 +18,12 @@ services:
...
       - .:/home/frontend/openqa-mail-notification
+  obs-package-status:
+    build:
+      dockerfile: Dockerfile
+      context: obs-package-status
+    restart: always
+    environment:
+      - RUN_EVERY=600
+    volumes:
+      - .:/home/package-bot/obs-package-status
```
Comment (DavidKang) [inline docker-compose.yml]: The volume directory should be: `/home/frontend/obs-package-status`, right?

Comment (danidoni) [reply docker-compose.yml]: Huh, yes. Changing.

**Pull request merged**